Classes/Controller/Backend/Order/OrderController.php

```diff
@@ -49,8 +55,10 @@ public function injectItemRepository(ItemRepository $itemRepository): void
 
     public function __construct(
         protected readonly ModuleTemplateFactory $moduleTemplateFactory,
-        protected readonly IconFactory $iconFactory
-    ) {}
+        protected readonly IconFactory           $iconFactory
```

Comment:
Can you remove the additional spaces here.

Reply:
Of course, sorry, that resulted from PhpStorm...